queue_services/entity-emailer/src/entity_emailer/email_templates/INVOL-DIS-STAGE-1.html

```diff
@@ -68,9 +68,14 @@
             {% if extra_provincials %}
             <p>
               Our records indicate your company is registered in 
-              {% for province in extra_provincials %}{{ province }}{% if not loop.last %} and {% endif %}{% endfor %} 
+              {% for province in extra_provincials %}
+              {{ province }}{% if not loop.last %}{% if loop.revindex == 2 %}, and {% else %}, {% endif %}{% endif %}
+              {% endfor %}
```

Comment:
Can you upload an example where the business only has 2 extraprovincial registrations? I think now it will generate 'X, and Y' instead of 'X and Y'. 

The wording in [design](https://www.figma.com/design/Tl8uDClgM5NAozzMltfiMD/Entities-Emails?node-id=2367-9&t=5EW1we50xRYerZwN-0) for your reference:
![image](https://github.com/user-attachments/assets/8579f7d8-5bd0-4e57-89e9-7ec5b894fbff)

Reply:
Great catch! Fixed now:
1 XPRO:
![1 item](https://github.com/user-attachments/assets/93600e02-e2de-490d-9a41-b097c777d201)

2 XPROS:
![2 items](https://github.com/user-attachments/assets/83ebcaf9-07c1-4838-b97e-277d66214a09)

3 or more XPROS:
![3 items](https://github.com/user-attachments/assets/a77db42c-10ff-4009-aadb-a58becee27f6)